packages/language-server/src/util.ts

```diff
@@ -525,3 +525,6 @@ export function getCompositeTypeFieldsRecursively(
     return getFieldsFromCurrentBlock(lines, typeBlock)
   }
 }
+
+// max value for i32
+export const getMaxSafeValue = (): number => Math.pow(2, 31) - 1
```

Comment:
I guess we could hardcode the value (2147483647) as const instead though that sounds like it would be a tiny optimization

Reply:
Fair! I wasn't sure what we'd prefer, can swap it

Reply:
Like you want, it's not that important

Reply:
Sorted c: